### ISSUE TITLE
css color variables don't work

### ISSUE THREAD

**Issue by Youngbae-Jeon** [primefaces/primereact#2166]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report
[ ] feature request
[ ] support request => Please do not submit support request here, instead see https://forum.primefaces.org/viewforum.php?f=57
```

On the latest update(primereact@6.4.1), I found that some css color variables don't work.
The vairables are defined using tint/shade functions like the followings...
```css
:root {
    --blue-50: tint(#2196F3, 95%);
    --blue-100: tint(#2196F3, 76%);
    --blue-200: tint(#2196F3, 57%);
    --blue-300: tint(#2196F3, 38%);
    --blue-400: tint(#2196F3, 19%);
    --blue-500: tint(#2196F3, 0%);
    --blue-600: shade(#2196F3, 15%);
    --blue-700: shade(#2196F3, 30%);
    --blue-800: shade(#2196F3, 45%);
    --blue-900: shade(#2196F3, 60%);
    ......
```
I'm not sure your css custom function definitions(tint/shade) are omitted or your css generator(like scss) didn't work properly.
Check it please. I'm using PrimeReact in my project with thankful mind to you.

**Comment by Youngbae-Jeon** [comment]
Additionally, before update, the variables were defined like this...
```css
:root {
    --blue-50: #f4fafe;
    --blue-100: #cae6fc;
    --blue-200: #a0d2fa;
    --blue-300: #75bef8;
    --blue-400: #4baaf5;
    --blue-500: #2196f3;
    --blue-600: #1c80cf;
    --blue-700: #1769aa;
    --blue-800: #125386;
    --blue-900: #0d3c61;
    ......
```
and they worked correctly.

**Comment by matthewmartinspok** [comment]
Also running into this issue. Would be nice for some clarification on this.

**Comment by matthewmartinspok** [comment]
Did some digging and found this:
https://github.com/primefaces/primereact/issues/2126
So seems like the RC versions might have this, but doesn't look like it's in the latest 6.4.1?
Shouldn't this have been a hotfix/patch to the latest build so that 6.4 isn't broken for everyone who uses these color vars?

**Comment by mertsincan** [comment]
Hi all,

Fixed for 6.5.0. I'll release it this week.

Best Regards,
